fern/definition/dnsrecords.yml

```diff
@@ -0,0 +1,22 @@
+types:
+  DnsRecord:
```

Comment:
Is there a reason not to put an ENUM type on DnsRecord and then have list<DNS_Records>?

Reply:
We spoke in person, but for anyone else seeing this, the fern definition here mimics the existing struct definitions that it replaces (see deleted code below), which align nicely with the way that the output is used downstream

Reply:
Agree that for now it makes sense to mirror the existing types to minimize churn. Can always migrate later